### DISCUSSION TITLE
Date picker - disable pop-up calendar on autofocus

### DISCUSSION THREAD

**Discussion by rchampagneca** [Laravel-Backpack/community-forum#356] [Q&A (Help)] [answered]
I've got a form with a date_picker as the very first field of a form. I wish to disable the automatic pop-up calendar on the form's entry and enable it only on field input. I've looked through the  [Bootstrap date_picker_options](https://bootstrap-datepicker.readthedocs.io/en/latest/) , but none seem to fit the bill. 

- Is there a Backpack attribute that can achieved this ? 
- Or, should I rely on a custom script ?

Regards.

**Comment by pxpm** [comment]
Hey @rchampagneca 

I think you are looking for: https://bootstrap-datepicker.readthedocs.io/en/latest/options.html#showonfocus

Let me know.

**Reply by rchampagneca** [reply]
Hello @pxpm,

I had tried the `'showOnFocus' => false` date_picker_option. However, it disables the pop-up calendar altogether.

**Reply by rchampagneca** [reply, marked as answer]
I've solved this using the config parameter in **/config/backpack/operations/update.php** :

// when the page loads, put the cursor on the first input?
    'autoFocusOnFirstField' => false,
